src/Attachment.php

```diff
@@ -203,10 +203,10 @@ public function path()
     public function src()
     {
         if (isset($this->abs_url)) {
-            return URLHelper::maybe_secure_url($this->abs_url);
+            return URLHelper::maybe_secure_url($this->abs_url) ?: null;
```

Comment:
Should we use coalesce operator instead? `??` instead of `?:`

Reply:
@xavivars I intentionally used `?:`, because `$this->abs_url` or the result of `URLHelper::maybe_secure_url()` will probably never be `null`. But it could be empty, in which case I want to return `null` instead of an empty string.

Reply:
Makes sense